avocado/utils/multipath.py

```diff
@@ -233,9 +236,10 @@ def is_failed():
 
 def reinstate_path(path):
     """
-    reinstating the individual paths
-    :param disk_path: disk path. Example: sda, sdb.
-    :return: True or False
+    Reinstate the individual paths.
+
+    :param path: disk path. Example: sda, sdb.
```

Comment:
We also use `:param str path: ...` one liners which seems to work in sphinx too but again it depends what your standards here are. Otherwise this should have another line with the path type.

Reply:
It is a nice suggestion, but I will change only in a few spots and leave other changes for the future.